TESTING.md

```diff
@@ -116,6 +116,37 @@ The `integration/happy/minimal` test is always a good start if you just want to
 
 In case you want to test the actually generated client, have a look at the `integration/happy/blog` test as an example.
 
+## How to trigger artificial panics
+
+Sometimes it may be useful to trigger a panic in the Rust binaries or libraries used by Prisma under the hood.
+Most of the Rust artifacts are shipped as binaries, whereas others (like `query-engine`) are shipped both as a library (by default) and as a binary (on demand).
```

Comment:
Only Query Engine is shipped as binary and library, so you can rephrase that a bit.

Reply:
Done